src/components/Tabber/utils.ts

```diff
@@ -3,22 +3,38 @@ import { SocialMedia } from '~services/api/social-media/social-media.types';
 import { SocialMediaState } from '~stores/useSocialMediaStore.types';
 
 import { Tab, TabId, Tabs } from './Tabber.types';
+import { AccountPost } from '~stores/useAccountStore';
 
 export const getFirstPostModeId = (
   socialMedia: SocialMedia
-): SocialMedia['postModes'][number]['id'] => socialMedia.postModes[0].id;
+): SocialMedia['postModes'][number]['id'] => {
+  if (
+    !socialMedia ||
+    !socialMedia.postModes ||
+    socialMedia.postModes.length === 0
+  ) {
```

Comment:
```suggestion
  if (socialMedia.postModes.length === 0
  ) {
```
São necessárias todas essas validações? tente assim, você testando se tem length no postmodes já resolve

Reply:
Estava testando só o básico mas temos muitas regras de lint etc... que estavam causando problemas, esta foi a solução que encontrei.